lib/modules/manager/vendir/__fixtures__/alias-contents.yaml

```diff
@@ -0,0 +1,30 @@
+---
+apiVersion: vendir.k14s.io/v1alpha1
+kind: Config
+directories:
+- path: vendor
+  contents:
+  - path: custom-repo-custom-version
+    helmChart:
+      name: oci-contour
+      version: "7.10.1"
+      repository:
+        url: alias:ociRegistry
```

Comment:
please use real values. this won't be detected as oci chart, because aliases are replaced after detect 

please use one simple inline test for aliases

Reply:
I pulled these values from [helmv3](https://github.com/renovatebot/renovate/blob/main/lib/modules/manager/helmv3/extract.spec.ts#L99-L116) I've updated the OCI one to be a real OCI test and ensure the replacement happens with the test.